package.json

```diff
@@ -15,7 +15,7 @@
     "watch:html": "npm run build:html && chokidar index.html -c 'npm run build:html'",
     "build:css" : "npm run clean:css && lessc --source-map css/index.less build/$npm_package_name.$npm_package_version.css",
     "watch:css" : "npm run build:css && chokidar 'css/**/*.less' -c 'npm run build:css'",
-    "clean:css" : "rm -f build/$npm_package_name.$npm_package_version.css build/$npm_package_name.$npm_package_version.map",
+    "clean:css" : "rm -f build/$npm_package_name.$npm_package_version.css build/$npm_package_name.$npm_package_version.css.map",
```

Comment:
I believe that if you don't have the `.css` here, it will cause an issue linking to the source map